JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardYourNetwork.kt

```diff
@@ -56,18 +57,18 @@ fun PostCardPopular(post: Post) {
                         Column(modifier = Spacing(16.dp)) {
                             Text(
                                 text = post.title,
-                                style = (+themeTextStyle { h6 }).withOpacity(0.87f),
+                                style = ((+MaterialTheme.typography()).h6).withOpacity(0.87f),
```

Comment:
Ditto with extracting typography above, no need to access the ambient value every time